src/hcpp-deque.c

```diff
@@ -39,108 +39,108 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hcpp-internal.h"
 #include "hcpp-atomics.h"
 
-void dequeInit(deque_t * deq, void * init_value) {
-	deq->head = 0;
-	deq->tail = 0;
+void deque_init(deque_t * deq, void * init_value) {
+    deq->head = 0;
+    deq->tail = 0;
 }
 
 /*
  * push an entry onto the tail of the deque
  */
-int dequePush(deque_t* deq, void* entry) {
-	int size = deq->tail - deq->head;
-	if (size == INIT_DEQUE_CAPACITY) { /* deque looks full */
-		/* may not grow the deque if some interleaving steal occur */
-		// std::cout<<getenv("PMI_RANK") <<": Deque full for worker-"<<current_ws()->id << std::endl;
-		// HASSERT("DEQUE full, increase deque's size " && 0);
-		return 0;
-	}
-	int n = (deq->tail) % INIT_DEQUE_CAPACITY;
-	deq->data[n] = (task_t*) entry;
-	deq->tail++;
-	return 1;
+int deque_push(deque_t* deq, void* entry) {
+    int size = deq->tail - deq->head;
+    if (size == INIT_DEQUE_CAPACITY) { /* deque looks full */
+        /* may not grow the deque if some interleaving steal occur */
+        // std::cout<<getenv("PMI_RANK") <<": Deque full for worker-"<<current_ws()->id << std::endl;
+        // HASSERT("DEQUE full, increase deque's size " && 0);
```

Comment:
Can we uncomment this assert? I think in production build, HASSERT doesn't do anything